src/utils/prepereDataFromServer.ts

```diff
@@ -0,0 +1,22 @@
+import { Person } from '../types';
+
+export const prepareDataFromServer = (dataFromServer: Person[]) => {
```

Comment:
🔥

Reply:
Thanks for all of your comments. ) it's not too much, fits perfect